### DISCUSSION TITLE
JavaScript error: Cannot read properties when resizing the window

### DISCUSSION THREAD

**Discussion by jeromehode** [datenstrom/community#993] [Report a bug] [answered]
minor bug: the following gets logged in the browser console when resizing the window with a "edit"-URL, editor not open
```
edit.js:1389 Uncaught TypeError: Cannot read properties of null (reading 'style')
    at Object.setOuterLeft (edit.js:1389:17)
    at Object.resizePane (edit.js:502:32)
    at onResizePane (edit.js:12:44)
```

**Comment by jeromehode** [comment]
quick fix: https://github.com/datenstrom/yellow/pull/994

**Comment by markseuffert** [comment, marked as answer]
Thank you for the bug report. The bug was probably introduced when the pane system was rewritten. I think the original system resized all panes when the browser window was resized, while the current system resizes only the current pane. No pane can be visible, paneID can become equal zero. The [edit extension](https://github.com/annaesvensson/yellow-edit) has been updated with an additional check as suggested.

Should be fixed now, please try it out.

**Reply by markseuffert** [reply]
What do you think, @jeromehode? Can you take a look and test the current solution?

**Reply by jeromehode** [reply]
tested: no more  TypeError in the console
